common/InternalRequestsCleaner.js

```diff
@@ -0,0 +1,21 @@
+const RequestsCleaner = require('./RequestsCleaner');
+
+class InternalRequestsCleaner extends RequestsCleaner {
+	verifySignatures() {
+		if (!this.calls) {
+			return;
+		}
+		Object.keys(this.calls).forEach((socketId) => {
```

Comment:
Consider using the babel polyfill for access to `Object.entries` and `Object.values`.